artemis/modules/nuclei.py

```diff
@@ -152,20 +152,21 @@ def run_multiple(self, tasks: List[Task]) -> None:
 
         links_per_task = {}
         links = []
-        for task in tasks:
-            links_per_task[task.uid] = self._get_links(get_target_url(task))
-            self.log.info("Links for %s: %s", get_target_url(task), links_per_task[task.uid])
-            links.extend(links_per_task[task.uid])
+        if not Config.Modules.Nuclei.NUCLEI_AUTOMATIC_SCAN:
```

Comment:
this is a separate topic, please keep the PRs separate

Reply:
yep sorry, the fix leaked to the main branch